tests/Unit/Provider/Youtube/ProviderTest.php

```diff
@@ -67,6 +67,11 @@ public function providesDataProvider()
             ['http://m.youtube.com/?feature=player_embedded&v=dQw4w9WgXcQ', true],
             ['http://m.youtube.com/v/dQw4w9WgXcQ?fs=1&hl=en_US', true],
             ['https://m.youtube-nocookie.com/embed/dQw4w9WgXcQ', true],
+			//test case for age restrictions (id: owUZUdyRVcQ)
```

Comment:
The data in this tests only test if the Youtube Provider recognices the url, so this urls arn't usefull in this case. But I will use the url for a manuel test of the PR.

Reply:
Would you like me to remove it then?